spec/system/ckeditor_spec.rb

```diff
@@ -51,4 +51,21 @@
     expect(page).not_to have_link "Upload"
     expect(page).not_to have_link "Browse Server"
   end
+
+  context "When navigating back to editor page using browser history back" do
+    scenario "display ckeditor unsaved contents", :js do
+      login_as(create(:administrator).user)
+
+      visit new_admin_newsletter_path
+      fill_in_ckeditor "Email content", with: "This is an unsaved body"
+      click_link "Newsletters"
+      expect(page).to have_link "New newsletter"
+      go_back
+
+      wait_for_ckeditor("Email content")
```

Comment:
I seems this line is needed on my local machine avoid the test to be stucked and make it pass (*), although to wait for ckeditor to be fully loaded has sense to me since we are destroying ckeditor before moving to Newsletter page and initializing it (again) when moving back to editor page, this re-initialization probably will take time to the browser and can cause flaky specs when ckeditor loading time exceeds the Capybara max wait time.

(*) Lately i am experiencing problems with latest versions of chrome and chromedriver so maybe this is only a problem of mine on macos OS. I am testing your changes to see if this is working as you explained in Travis which is also a Linux environment like yours.